### DISCUSSION TITLE
【项目组队】肉搏小游戏

### DISCUSSION THREAD

**Discussion by wow-sven** [creatorsand/.github#32] [Ideas]
项目名称：肉搏小游戏

项目描述：

1. 前10/5分钟或其他触发条件(玩家等级达到满级)为发育阶段，可以杀一些野怪或者boos，拿取奖励（装备、经验、技能书？）。
2. 发育完成后，开启玩家pvp，选出最后赢家。

项目发起人：sven

其他角色需求：全栈\合约

有无Bounty：无

界面表现参考：https://mud.dev/tutorials/emojimon/

**Comment by ShirleneLiu** [comment]
我感兴趣参加，我是后端
